Source/WebKit/UIProcess/glib/DisplayVBlankMonitorDRM.cpp

```diff
@@ -182,6 +182,45 @@ static std::optional<std::pair<uint32_t, uint32_t>> findCrtc(int fd)
 }
 #endif
 
+using DrmNode = UnixFileDescriptor;
+using CrtcInfo = std::pair<uint32_t, uint32_t>;
+#if PLATFORM(GTK) || (PLATFORM(WPE) && ENABLE(WPE_PLATFORM))
+static std::optional<std::pair<DrmNode, CrtcInfo>> findDrmNodeWithCrtc(PlatformMonitor* monitor = nullptr)
+#else
+static std::optional<std::pair<DrmNode, CrtcInfo>> findDrmNodeWithCrtc()
+#endif
+{
+    constexpr int maxDrmDevices = 256;
```

Comment:
This is not the maximum number of devices, but the maximum nodes (3x64 rounded to 256), but here we want the maximum number of devices including 3 possible nodes, so it should be 64.